exporter/prometheusexporter/end_to_end_test.go

```diff
@@ -106,7 +107,7 @@ func TestEndToEndSummarySupport(t *testing.T) {
 		PrometheusConfig: receiverConfig,
 		ReceiverSettings: config.NewReceiverSettings(config.NewComponentID("prometheus")),
 	}
-	// 3.5 Create the Prometheus receiver and pass in the preivously created Prometheus exporter.
+	// 3.5 Create the Prometheus receiver and pass in the previously created Prometheus exporter.
```

Comment:
Any idea why we need a prometheus receiver here?  We don't seem to do anything with it, and the extra calls it makes seem disruptive to the rest of the test.

Reply:
The receiver handles scraping the mocked exposition and converting it to pdata, then passing it to the exporter.